### PR TITLE
change sub path of the package url

### PR DESCRIPTION
**Pull request by liyakun**
In the [pull request 34](https://github.com/openspending/os-conductor/pull/34), the modification to the returned package URL path is not enough, we also need to remove the sub path '/final/' from the returned URL.

### CODE REVIEW

**Comment by liyakun** [comment]
@brew Sorry for the inconvenience, I did not realized that we also need to remove the '/final/' part from the returned package URL. I added another regex expression to remove it in this PR. Thanks.

File: conductor/blueprints/package/controllers.py [hunk 1]
```diff
@@ -301,11 +302,13 @@ def run_hooks(name, token, pipeline):
...
         return None
     _, datapackage_url, _, _, _, _, _, _ = package_registry.get_raw(name)
-    # Fix datastore.openspending.org url
+    
+    # Fix datastore.openspending.org url    
     datapackage_url = \
         re.sub(r'https?://datastore\.openspending\.org/',
                'https://s3.amazonaws.com/datastore.openspending.org/',
-               datapackage_url)
+               re.sub(r'/final/datapackage.json', '/datapackage.json', datapackage_url))
```
Comment (brew) [inline conductor/blueprints/package/controllers.py]: I think you should escape the period in `r'/final/datapackage.json'`.

**Comment by liyakun** [comment]
@brew Thanks, I have made the period escape and solved linting issues.

**Comment by brew** [comment]
This should be deployed now.

**Pull request merged**